lisa/tools/git.py

```diff
@@ -91,6 +91,9 @@ def checkout(
             # create a temp branch to checkout tag or commit.
             checkout_branch = f"{constants.RUN_ID}"
 
+        # mark directory safe
+        self._mark_safe(cwd)
```

Comment:
When is it needed? Why it's in `checkout` only? If the code is checked out multiple times, it shouldn't run multiple times. So this command should be have a flag to run only once.